c/meterpreter/source/extensions/stdapi/server/sys/process/process.c

```diff
@@ -260,6 +261,33 @@ DWORD request_sys_process_execute(Remote *remote, Packet *packet)
 		if (flags & PROCESS_EXECUTE_FLAG_SUSPENDED)
 			createFlags |= CREATE_SUSPENDED;
 
+		if (TRUE) {

```

Comment:
Does this serve any purpose other than to delineate the code block?

Reply:
yes, it was meant only meant to debug the agent implementation since the integration with msf backend is still missing.